core/server/server.js

```diff
@@ -201,7 +195,7 @@ class Server {
     })
 
     if (!rasterUrl) {
```

Comment:
I believe this answers the question I asked in the main PR thread regarding raster proxies, correct?